lib/game.rb

```diff
@@ -29,46 +31,48 @@ def set_options
       @human_player.marker = @marker
     end
 
+    def generate_markers
+      markers = []
+
+      @players.each do |player|
+        markers << player.marker
+      end
+
+      markers
+    end
+
     def run_game_loop
-      until is_game_over?
+      markers = generate_markers
+
+      until TicTacToe::is_game_over?(@board, markers)
         if is_computer_the_current_player?
```

Comment:
^^
To add on to that - I remember that we talked about this on Friday, so this may still be an "in progress" task for you, Malina - but imagine if we asked you to have an option where two computers play. This would mean you'd have to change a lot of the logic in `Game` as you've already hard coded an expectation that there is 1 computer and 1 human playing.